package.json

```diff
@@ -38,6 +38,7 @@
     "ffjavascript": "^0.2.33",
     "js-sha3": "^0.8.0",
     "level": "^6.0.1",
-    "web3-utils": "^1.2.11"
+    "web3-utils": "^1.2.11",
+    "@hermeznetwork/commonjs-old": "git+https://github.com/hermeznetwork/commonjs.git#master"
```

Comment:
eventually, the new version with the new rollupDB will reach master and a npm package. I think it is better to point the current npm version of commonjs here: https://www.npmjs.com/package/@hermeznetwork/commonjs/v/0.3.3